qadence/backends/pulser/backend.py

```diff
@@ -335,8 +336,9 @@ def expectation(
             mitigation_fn = mitigation.get_mitigation_fn()
             mitigated_exp_val = mitigation_fn(
                 backend_name=self.name,
-                circuit=circuit,
-                observable=observables,
+                circuit=circuit.original,
```

Comment:
why original and not abstract?

Reply:
Mainly because all protocols so far require to run some experiments with the original circuit and observable and parameters. They get re-embedded and re-converted in another instance of the backend. This is also one more reason for outsourcing protocols.